### DISCUSSION TITLE
Implement INOL

### DISCUSSION THREAD

**Discussion by lsolesen** [astashov/liftosaur#188] [Ideas]
Right now number of sets are used in the workout planner as a means of gauging how tuff the workout plan is.

However, when doing reps with lower intensity that overestimates how hard it is.

INOL could be another way to know how hard the planned workload actually is.

https://liftvault.com/resources/inol-calculator-tool/

**Comment by yrinkel** [comment]
Oh that’s actually quite interesting. Finally a rating that takes in account the number of sets!
